src/modules/Requests.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const Requests = {
+  async searchSelfCare(searchQuery) {
+    const response = await axios.get(`/api/search?q=${searchQuery}`, {
+      headers: { API_KEY: process.env.REACT_APP_API_KEY },
+    });
+    return response
+  },
+};
+
+export default Requests;
```

Comment:
For modules, I would prefer if we stick to CRUD convention that uses the resource name. So this module should be named `Search` and method `create<´ since we are making a POST (corresponding to the methods on the API side.

Reply:
changed